File: 1-easy/problems/1512-number-of-good-pairs/solution.ts
function numIdenticalPairs(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: int
  """
  return sum(c*(c-1)//2 for c in collections.Counter(nums).itervalues());
}

export { numIdenticalPairs };

File: 1-easy/problems/1512-number-of-good-pairs/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def numIdenticalPairs(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        return sum(c*(c-1)//2 for c in collections.Counter(nums).itervalues())
